README.md

```diff
@@ -6,38 +6,81 @@ the Transmission BitTorrent client, using its HTTP RPC protocol.
 
 # DEPENDENCIES
 
-The following packages are required dependencies (debian/ubuntu).
+### Runtime Required Dependencies
+The following packages are required dependencies:
 
-```bash
-libgtk-3-dev automake autoconf gcc libgeoip-dev gettext autoconf-archive libappstream-dev appstream-util libcurl4-openssl-dev libjson-glib-dev
+ - gtk >= 3.16
+ - glib >= 2.44 (including gio and gthread)
+ - json-glib >= 0.8
+ - libcurl
+ - GNU gettext
+
+An example command for getting the packages required for an Ubuntu/Debian system:
+
+```
+# apt-get install libgtk-3-dev libgeoip-dev gettext libcurl4-openssl-dev libjson-glib-dev
 ```
 
-# BUILDING
+### Runtime Optional Dependencies
+The following packages are optinal dependencies
+
+ - libmrss >= 0.18, RSS feed support
+ - libproxy, HTTP proxy support
+ - libgeoip, country of origin of peers
+ - libnotify, desktop notifications support
+ - libappindicator, Application tray support
 
-Optionally install `libgeoip` and its headers to see the country
-origin of peers.
+### Build Dependencies
+Transmission Remote Gtk uses meson for its build system. A relatively new version of
+meson is required.
 
+ - meson >= 0.59.0
+ - ninja
+ - gcc >= 4.5
```

Comment:
```suggestion
```

Frankly nobody tests what version of C compiler we need, this version is 12 years old, old enough to learn C itself. Any version of clang or gcc people use will work fine, this is a boring codebase.

Reply:
I actually just googled "minimum gcc version for C99" but I do agree that's a ridiculous requirement. If someone with an old stable openBSD comes around or something they know what they're getting into.